basics/plugin_structure/plugin_icon_file.md

```diff
@@ -1,42 +1,93 @@
-size icon_ should be supplied because it will automatically
-scale.
+## Introduction
+Plugin Logos are shown in the [Plugins Repository](https://plugins.jetbrains.com/), and in [MarketPlace](https://plugins.jetbrains.com/marketplace).
+They also appear in the Settings/Preferences [Plugin Manager](https://www.jetbrains.com/help/idea/managing-plugins.html) UI in JetBrains' IntelliJ Platform-based IDEs. 
+Whether online or in the product UI, a Plugin Logo helps users to identify a plugin more quickly in a list, as shown below:
 
-| Icon Size  | Example SVG Icon                            |
-| ------------- | -------------                            |
-| 40x40         | ![Kotlin Plugin Icon](img/kotlin40.svg)  |
+![Example Product Plugin Preferences Dialog](img/plugin_prefs.png){:width="800px"}
 
-### Plugin Icon File Naming Convention
-Plugin Icon files are named according to the following conventions:
-* `pluginIcon.svg` for use with the default (light) JetBrains IDE theme
-* `pluginIcon_dark.svg` for use with the Darcula JetBrains IDE theme
+## Plugin Logo Icon Requirements
+Plugin Logos are represented as icons, and to be displayed correctly within an IntelliJ Platform-based IDE the icon must:
+* Be the correct file format,
+* Follow the best practices design guidelines
+* Conform to file name conventions.  
 
-### Adding Plugin Icons to a Plugin Project
-The Plugin Icon files must be in the `META-INF` folder of the plugin distribution file, 
-i.e. the `*.jar` or `*.zip` file you upload to the plugin repository and install into
-a JetBrains IDE.
+### Plugin Logo Icon File Format
+All Plugin Logo images must be SVG format. 
+This vector image format is required because the Plugin Logo is displayed in more than one size.
 
-To include Plugin Icons in your distribution file, place the Plugin Icon files 
-into a plugin project's `resources/META-INF` folder. Note this requirement
+### Plugin Logo Size
+The Plugin Logo icon should be provided in one size: 40px x 40px. 
+However, the icons display in two sizes, and scale automatically in each context:
+* 40px by 40px in the plugins _list_ in the Plugin Manager UI.
+* 80px by 80px in the plugin _details screen_ in the Plugin Manager UI, on the plugin's page in the Plugins Repository, and on MarketPlace.
+
+Verify that Plugin Logo icon designs are effective in both sizes and all display contexts.
+
+### Plugin Logo Shape and Visual Weight
+The nominal Plugin Logo size is shown below. 
+At least 2px transparent padding should remain around the perimeter. 
+This leaves a nominal 36px x 36px area where the visible part of the Plugin Logo icon should fit, as shown below:
+
+![36px x 36px is the area where the visible part of the logo icon should fit](img/icon_size.png){:width="225px"}
+
+Make sure Plugin Logos have the same visual weight as the logos in the examples below. 
+The more visual weight in Plugin Logo design, the less actual space it needs. 
```

Comment:
Apologies. I changed the wording because I did not think "...more filled an icon is" was clearer for users. When in doubt, I think it's better to use a term of art.

I referred to the [IntelliJ Platform UI Guide for Icons](https://jetbrains.design/intellij/principles/icons/#08) which uses the term "visual weight". I'm assuming the audience would be the same for this section and that spec?